lib/app/views/default.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container" role='main'>
   <div id="header-module-name">
   <%=instance.module.title%>
-  <% if instance.module.title != "SMART Application Launch Framework Implementation Guide v1.0.0" %>
+  <% if instance.module.fhir_version != nil %>
```

Comment:
`#nil?` would be more Ruby idomatic

i.e. `unless instance.module.fhir_version.nil?`

[#nil?](https://ruby-doc.org/core-2.6.3/Object.html#method-i-nil-3F)
[#unless](https://ruby-doc.org/docs/keywords/1.9/Object.html#method-i-unless)